README.md

```diff
@@ -26,21 +26,22 @@ Make sure you have these environment variables set:
 ```
 DATABASE_URL=postgres://USERNAME:PASSWORD@HOST:PORT/DBNAME
 SECRET_KEY='abcd1234'
+LAND_AVAILABILITY_API_URL=http://localhost:8000
```

Comment:
let's keep in mind that in case someone wants to setup a complete development environment on their own machine, different apps/services will have to run on different ports. That's why I used 8000, 8001, 8002 etc...